Tests/SwiftParserTest/translated/TryTests.swift

```diff
@@ -234,8 +266,8 @@ final class TryTests: XCTestCase {
       """,
       diagnostics: [
         DiagnosticSpec(locationMarker: "1️⃣", message: "'try' must be placed on the thrown expression", fixIts: ["move 'try' after 'throw'"]),
-        DiagnosticSpec(locationMarker: "2️⃣", message: "expected expression in 'try' expression"),
-      ], fixedSource: "throw try <#expression#>"
+        DiagnosticSpec(locationMarker: "2️⃣", message: "expected expression after 'try'"),
+      ], fixedSource: "throw try"
```

Comment:
I actually like the placeholder here, though do prefer the "after" rather than "in ... expression".

Reply:
I thought it provider very little value. But I don’t have strong opinions either, so I’m adding it back in.